src/advanced/bitcoin-anchoring.md

```diff
@@ -70,65 +69,42 @@ floor(2/3*N) + 1` is used as a supermajority.
-even if the previous one has not been included yet in any Bitcoin block.
-However, there is a chance that this previous transaction or one of its
-ancestors was mutated by a malicious validator as described above, and the
-mutated transaction has been committed to the Bitcoin blockchain. This
-would make the original transaction ineligible for inclusion into the
-Bitcoin blockchain.
-
-To handle this problem, the consensus is used to select the appropriate previous
-transaction via [LECTs](#lect).
-
-### LECT
-
-Every validator defines which anchoring transaction is considered to be
-the latest one. This transaction should be spent in the new anchoring
-transaction and is called the Latest Expected
-Correct Transaction (LECT). LECTs of all validators are published in the
-Exonum blockchain. While creating a new anchoring transaction, the
-supermajority of validators select a common LECT and spend its change output.
-
-Every validator refreshes its LECT with a [custom
-schedule](#lect-updating-interval). To get a new LECT, the validator uses the
-API of a [Bitcoin node](#bitcoind-node). The new LECT must have the following
-properties:
-
-- It is a valid anchoring transaction for the current Exonum blockchain;
-- It may have any amount of confirmations, in particular, 0;
-- Its change output should be unspent. That means that the specified
-  validator believes there was no following anchoring transactions after
-  this one;
-- Among all Bitcoin transactions satisfying the previous properties,
-  LECT should have the greatest anchored Exonum block height;
-- If multiple transactions meet the previous requirements, any of them
-  may be chosen as the LECT.
-
-The LECT solves the transaction malleability problem, though anchoring
-transactions sometimes may be orphaned and never written to the Bitcoin
-blockchain. However, it is safe enough as the following anchoring
-transactions effectively anchor all previous ones.
-
-Exonum uses a [`bitcoind` client](#bitcoind-node), and only one of the
-transactions that meet the requirements can be considered valid by the
-bitcoind-node.
+than `M` signatures can be published (it is a common case). Previously, as these
+signatures formed a part of the anchoring transaction body, the transaction hash
+depended on the number and order of signatures put over said transaction.
+Therefore, the anchoring service faced a problem of
+[transaction malleability][transaction_malleability] -
+there was a chance that an anchoring transaction could be mutated by some
+malicious validator. The validator
+would use a different set of signatures for the same transaction and commit it
+to the Bitcoin network. This would make the original transaction ineligible for
+inclusion into the Bitcoin blockchain.
+
+As of Exonum 0.10 version the problem was solved by introducing
+[Segregated Witness][segwit] into the
+anchoring service. This allowed to store signatures of the anchoring
```

Comment:
> This allowed to store signatures...

I don't think I understand this sentence. It's not immediately clear whether it concerns the Exonum blockchain or Bitcoin; first time I've read it, I've interpreted it as concerning Exonum (after all, _anchoring_ transactions are singled out) and was quite confused. IMO, it could make sense to skip this sentence altogether.